docker-compose.selenium-chrome.yaml

```diff
@@ -7,7 +7,7 @@
 version: '3.6'
 services:
   selenium-chrome:
-    image: seleniarm/standalone-chromium:4.1.4-20220429
+    build: selenium-standalone-chrome
```

Comment:
This just says build using the Dockerfile in the selenium-standalone-chrome directory.